packages/ipfs-grpc-protocol/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
It's needed by the modules that depend on this one, otherwise you get:

```
ipfs-grpc-server: error TS5083: Cannot read file '/Users/alex/Documents/Workspaces/ipfs/js-ipfs/packages/ipfs-grpc-protocol/tsconfig.json'.
ipfs-grpc-server: npm info lifecycle ipfs-grpc-server@0.0.0~build:types: Failed to exec build:types script
ipfs-grpc-server: npm ERR! code ELIFECYCLE
ipfs-grpc-server: npm ERR! errno 1
ipfs-grpc-server: npm ERR! ipfs-grpc-server@0.0.0 build:types: `tsc --build`
ipfs-grpc-server: npm ERR! Exit status 1
ipfs-grpc-server: npm ERR! 
ipfs-grpc-server: npm ERR! Failed at the ipfs-grpc-server@0.0.0 build:types script.
```

Reply:
Actually I think it's because I've defined references to the protocol module in the server & client tsconfig.json files, maybe it's not necessary after all.